### PR TITLE
Compatibility with documentation generator

### PR DESCRIPTION
**Pull request by Swader**
Modified the README to be usable and readable both here but also when parsed by the doc generator. The generator will now use the content here to build the documentation at nimbus-libs.status.im, and it will then also generate API reference docs, adding those under a separate menu. The individual submodule MD files are currently little more than placeholders with basic description - they are meant to be extended with usage examples.

### CODE REVIEW

File: docs/sysrand.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Sysrand
+
+This module implements interface to operation system's random number generator.
+
+**Windows** using BCryptGenRandom (if available), CryptGenRandom(PROV\_INTEL\_SEC) (if available), RtlGenRandom.
```
Comment (cheatfate) [inline docs/sysrand.md]: You have some symbol conversion bug here must be `PROV_INTEL_SEC`.

File: README.md [hunk 2]
```diff
@@ -21,4 +24,91 @@ echo keccak_256.digest("Alice makes a hash")
 # outputs EF0CC652868DF797522FB1D5A39E58E069154D9E47E5D7DB288B7200DB6EDFEE
 ```
 
-See full documentation [here](https://cheatfate.github.io/nimcrypto).
\ No newline at end of file
+## Docs
```
Comment (cheatfate) [inline README.md]: I really do not like "Docs" maybe it would be better to have "Documentation"?

File: README.md [hunk 3]
```diff
@@ -2,11 +2,14 @@
 
 # Nim cryptographic library
 
+See documentation at [Nimbus Libs](https://nimbus-libs.status.im).
```
Comment (cheatfate) [inline README.md]: Please put this link under `Docs` as alternative option.

Comment (Swader) [reply README.md]: Better?

File: README.md (hunk 2 above)
Comment (Swader) [reply README.md]: Done

**Pull request closed without merging**